package.json

```diff
@@ -49,6 +56,7 @@
     "jscpd": "^3.5.9",
     "lint-staged": "^13.2.3",
     "madge": "^6.1.0",
+    "shx": "^0.3.4",
     "stop-only": "^3.3.1",
```

Comment:
`shx` can probably be removed.

Future PR will probably swap out `ts-node` for `node` with a transpile/dist step. Not sure `js` consumers could use this directly otherwise.

Reply:
If `shx` can be removed, any reason to not do so as part of the initial PR?

Reply:
Just didn't notice it had snuck in